src/mod/applications/mod_avmd/avmd_fast_acosf.c

```diff
@@ -176,6 +177,7 @@ extern int init_fast_acosf(void)
 	acos_table = (float *) mmap(
 			NULL,							   /* kernel chooses the address at which to create the mapping */
 			ACOS_TABLE_LENGTH * sizeof(float), PROT_READ, MAP_SHARED, acos_fd, 0);
+	fclose(acos_fp);
```

Comment:
This is closed in `destroy_fast_acosf()`

Reply:
acos_fd is closed in `destroy_fast_acosf` and `acos_fp` is local to `init_fast_acosf()`